### ISSUE TITLE
Paola comments (transfer)

### ISSUE THREAD

**Issue by shawntanzk** [cell-ontology/cell-ontology.github.io#6]
Moving over from old repo

> Could you please increase the font size of anything other than the section titles?

- [ ] should be able to, will look into this. To fix

> "The Cell Ontology (CL) is an OBO Foundry ontology covering the domain of canonical, natural biological cell types." => canonical, natural biological feels a little redundant. How about "...domain of natural cell types" or "physiological" or "physiologically-occurring". Possibly "The Cell Ontology (CL) is an OBO Foundry ontology covering the domain of physiological animal cell types.". Yes there are a few non-metazoan cell types in there, but the focus is on Metazoa.

- [ ] I took this from @addiehl 2016 paper - would want his input before changing this. 

> The funders' icons are apparently hyperlinked, but the redirect seems inactive.

- [ ] to fix, ideally removing hyper link capability (don't think its needed)

> "Browse The Cell Ontology - powered by the Ontology Lookup Service (OLS)" should probably read "...powered by the EMBL-EBI's Ontology...". True, EBI is acknowledged on the page, but it's a generic acknowledgement. Mention of OLS in the Ontology Tools Integrated section should also quote EMBL-EBI.

- [x] to fix

> In the FAIR principles section, Gene ontology should read Gene Ontology please.

- [x] to fix

> The GO FAIR screenshot and all others are not legible or linkable - are they meant to be?

- Defs not linkable, I prefer it not to be fully legible as that's not the point - it's just eye candy more than anything. 

> Could the phrasing in the Active Community section be improved please.

- [ ] will try lol, not sure if I can >.<

> cell ontology should read Cell Ontology (Widely Compatible and Applications sections)

@matentzn pointed this out too - fixed now

> Links under Applications (e.g. BICCN, HubMap) and Contact Us (e.g. GitHub) open in the same page. Personally I'd prefer it if they opened in new pages.

- [x] to fix

**Comment by addiehl** [comment]
Keep in mind that CL:cell (and eventually COB:cell) is a class intended to be the parent of any biological cell. By agreement with various communities, the development of the CL has focused on animal cell types, but CL retains classes such as 'cell in vitro' and 'abnormal cell' and their small hierarchies to allow for better interoperability with other cell ontologies, such as the Cell Line Ontology and the Cancer Cell Ontology, for instance.  Also, not all metazoan cell domains are curated via CL, such as fruitfly cells and plant cells.

How about:
The Cell Ontology (CL) is an OBO Foundry ontology covering the domain of biological cell types with curation focused on animal cell types and interoperability with specialized ontologies of cell types in other areas of biology.

A bit wordy, perhaps, but not wrong.

**Comment by shawntanzk** [comment]
Thanks @addiehl - will make the change
